src/main/java/pages/user/UserPageContainer.java

```diff
@@ -1,27 +1,32 @@
 package main.java.pages.user;
 
+import main.java.pages.header.HeaderFull;
 import main.java.tools.WebDriver;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
 /**
  * 
  */
-public class UserPageContainer {
+public class UserPageContainer extends HeaderFull {
 
-	private String actualPageName;
+	private WebElement actualPageName;
 	private WebElement pageName;
 	private WebElement address;
 	
     /**
      * Default constructor
      */
     public UserPageContainer() {
+    	super();
     	// pages.User page elements initialization
-    	actualPageName = WebDriver.driver.findElement(By.xpath("//*[@id='main']/header/h1/text()")).getText();
+	    //We will get text from actualPageName later in test
+    	actualPageName = WebDriver.driver.findElement(By.xpath("//*[@id='main']/header/h1"));
     	//Assert.assertTrue(actualPageName.contains("Your account"));
-    	pageName = WebDriver.driver.findElement(By.xpath("//*[@id='main']/header/h1/text()"));
-    	address = WebDriver.driver.findElement(By.xpath("//a[@id='addresses-link']/span"));
+	    //We will get text from pageName later in test
+    	pageName = WebDriver.driver.findElement(By.xpath("//*[@id='main']/header/h1"));
+    	//Don`t understand what actually you search
```

Comment:
maybe  better init fields in separate method?

Reply:
it is Nata`s method so i just changed minimum